src/AppInstallerCommonCore/AppInstallerTelemetry.cpp

```diff
@@ -35,6 +35,10 @@ namespace AppInstaller::Logging
         // Used to disable telemetry on the fly.
         std::atomic_bool s_isTelemetryEnabled{ true };
 
+        std::string s_executionStage;
```

Comment:
>std::string s_executionStage; [](start = 8, length = 29)

I think that while logging a string will be more flexible for adding new stages, it will make processing that data far harder.  Instead, we should just log an integer, and the current stages can be ParseArgs = 1000, Discovery = 2000, etc.